mojap_metadata/extractors/postgres_metadata.py

```diff
@@ -0,0 +1,99 @@
+import warnings
+from collections import defaultdict
+
+from mojap_metadata import Metadata
+import mojap_metadata.extractors.postgres_functions as pg
+
+
+def convert_to_mojap_type(ct: str) -> str:
+    """Converts our postgress datatypes to mojap-metadata types
+
+    Args:
+        ct (str): str representation of postgres column types
+
+    Returns:
+        str: String representation of our metadata column types
+    """
+
+    if ct in ["int8", "bigint"]:
+        t = "int64"
+    elif ct in ["int2", "int4", "integer", "smallint"]:
+        t = "int32"
```

Comment:
Would you not want your types to match those that arrow have? I am pretty sure arrow and postgres have pretty solid overlap for integers i.e. postgres `int8` = arrow `int8`
https://arrow.apache.org/docs/python/api/datatypes.html

Reply:
done